### PR TITLE
doc: 📝 V1 getting started with AI Dashboard

### PR DESCRIPTION
**Pull request by philippart-s**
Guide to explain the role of the new AI Dashboard

### CODE REVIEW

**Comment by eleapttn** [comment]
@philippart-s thanks for the doc 🙂!

Some comments (but this is just my point of view):
- In section *Introduction - first image*: is it possible to add a "complete" image of the dashboard, and not a cropped one?
- In section *Introduction* or *Dashboard tab*: is it possible to talk about the random suggested tutorials on the first dashboard page? Or just add it on a screenshot?
- *Open-ended question*: should we specify somewhere (in a primary for example) that the dashboard is not an AI product but just a global management interface?

Otherwise all is good for me 👍

**Comment by philippart-s** [comment]
Thanks for your feedback @eleapttn, here my answers:

 
>     * In section _Introduction - first image_: is it possible to add a "complete" image of the dashboard, and not a cropped one?

Yes I can, I wanted juste to put a small image to be less impressive ;), but ok

>     * In section _Introduction_ or _Dashboard tab_: is it possible to talk about the random suggested tutorials on the first dashboard page? Or just add it on a screenshot?

I have a dedicated explanation at the end of the guide, is not enough? 

>     * _Open-ended question_: should we specify somewhere (in a primary for example) that the dashboard is not an AI product but just a global management interface?

I did that at the beginning of the guide: https://github.com/ovh/docs/blob/adfd7cbba488f56687a20d933dca349300ef5cd7/pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md?plain=1#L25

You prefer that I use a Primary formatting for this part ?

File: pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+## Objective
+
+OVHcloud provides a set of managed AI tools designed for building your machine learning projects.
+
+This guide explains how to get started with OVHcloud AI Dashboard, a consolidated view of your usages of the OVHcloud AI tools.
+
+## Requirements
+
+- a [Public Cloud project](https://www.ovhcloud.com/en-gb/public-cloud/) in your OVHcloud account
+- access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB)
```
Comment (mhrng) [inline pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md]: I'd switch the order of both bullet points in a logical order (cf other AI guides)
```
- Access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB)
- A [Public Cloud project](https://www.ovhcloud.com/en-gb/public-cloud/) in your OVHcloud account
 ```

File: pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Introduction
+
+**AI Dashboard** is not a product but an overview of all AI products you use, billing information, security information and guide (like this one) that you can read.
```
Comment (mhrng) [inline pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md]: Suggested modifications: 
```
**AI Dashboard** is not a product but an overview of the AI Tools you use as well as an interface where you will find all transversal items such as security access, CLI and billing information, or guides (like this one).
```

File: pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+It's accessible via the main menu `AI & Machine Learning` in the `Public Cloud` universe.
```
Comment (mhrng) [inline pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md]: Je crois que l'on utilise peu la contraction d'habitude :) 

`It is`

File: pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+![Dashboard menu](images/dashboard-01.png){.thumbnail}
+
+The following chapters will help you to discover each section of the `AI Dashboard`.
+
+## Dashboard tab
+
+![Dashboard tab](images/dashboard-02.png){.thumbnail}
+
+In this section you will find all information related to your use of the AI Tools: [Object Storage](https://docs.ovh.com/gb/en/storage/), [AI Notebooks](https://docs.ovh.com/gb/en/publiccloud/ai/notebooks/definition/), [AI Training](https://docs.ovh.com/gb/en/publiccloud/ai/training/submit-job/) and [AI Deploy](https://docs.ovh.com/gb/en/publiccloud/ai/deploy/getting-started/).
```
Comment (mhrng) [inline pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md]: On ne met pas beaucoup d'infos sur les object storages pour l'instant, du coup pour éviter la confusion, on pourrait plutôt mettre :
```
In this section, you will find information related to your use of the AI Tools with easy access to [AI Notebooks](https://docs.ovh.com/gb/en/publiccloud/ai/notebooks/definition/), [AI Training](https://docs.ovh.com/gb/en/publiccloud/ai/training/submit-job/) and [AI Deploy](https://docs.ovh.com/gb/en/publiccloud/ai/deploy/getting-started/).
Listed instances are the ones that are in `RUNNING` and `STOPPED` status.

You can also easily access to your [Object Storages](https://docs.ovh.com/gb/en/storage/).
```

File: pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+At a glance, you can see the total amount of your next billing for your consumption with the AI tools.
+You can see more details on the prices of the AI tools in the [documentation](https://www.ovhcloud.com/en-gb/public-cloud/prices/#ai-&-machine-learning).
```
Comment (mhrng) [inline pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md]: `You can find more details on the prices of the AI tools [here](https://www.ovhcloud.com/en-gb/public-cloud/prices/#ai-&-machine-learning).`

File: pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+You can also see some security information about the number of created users (and tokens) that can use your AI tools.
```
Comment (mhrng) [inline pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md]: `You can also see security information with the number of users and tokens created to use your AI Tools.`

File: pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+## Users & tokens
+
+![Users & tokens tab](images/dashboard-03.png){.thumbnail}
+
+In this section you will find all information about the users and tokens you have created.
+You can also manage them (create, update and delete).
+To know how to manage your users and token, please read the guide Manage [AI users and roles](https://docs.ovh.com/gb/en/publiccloud/ai/users/).
+
```
Comment (mhrng) [inline pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md]:
```
In this section, you will find all information about your AI users and tokens.
You can easily create new Public Cloud users with dedicated AI Tools roles (`AI Platform - Operator` or `AI Platform - Read-Only`) and access to all your Public Cloud users.

You can also manage tokens (create, update and delete) that you can then use with your notebooks, jobs or apps.

To know how to manage your users and tokens, please read the guide [Manage AI users and roles](https://docs.ovh.com/gb/en/publiccloud/ai/users/).
```

File: pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+## Command Line Interface 
+
+![Command Line Interface](images/dashboard-04.png){.thumbnail}
+
+In this section you will find all information about how to install and use our CLI.
+You have also a list of the useful existing guides on the CLI.
```
Comment (mhrng) [inline pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md]:
```
In this section, you will find all information about how to install, authenticate and use our CLI.
You will also find a list of all useful guides on the CLI.
```

File: pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+
+## Documentation
+
+![Documentation ](images/dashboard-05.png){.thumbnail}
+
+In this section you will find a random list of our guides about AI products.
+You can refresh the list by clicking on the round arrow.
```
Comment (mhrng) [inline pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md]: `In this section, you will find a list of our guides about AI products. The list is on shuffle, you can refresh it to get other tutorials by clicking on the round arrow.`

**Review by eleapttn — APPROVED** [review]
🚀

**Review by mhrng — CHANGES_REQUESTED** [review]

File: pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md (hunk 1 above)
Comment (philippart-s) [reply pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md]: done with [917d6f9](https://github.com/ovh/docs/pull/4373/commits/917d6f95d3e0f003df60f7a0c0e63eeb7ed5ce2b)

File: pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md (hunk 2 above)
Comment (philippart-s) [reply pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md]: done with [917d6f9](https://github.com/ovh/docs/pull/4373/commits/917d6f95d3e0f003df60f7a0c0e63eeb7ed5ce2b)

File: pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md (hunk 4 above)
Comment (philippart-s) [reply pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md]: done with [917d6f9](https://github.com/ovh/docs/pull/4373/commits/917d6f95d3e0f003df60f7a0c0e63eeb7ed5ce2b)

File: pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md (hunk 5 above)
Comment (philippart-s) [reply pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md]: done with [917d6f9](https://github.com/ovh/docs/pull/4373/commits/917d6f95d3e0f003df60f7a0c0e63eeb7ed5ce2b)

File: pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md (hunk 7 above)
Comment (philippart-s) [reply pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md]: done with [917d6f9](https://github.com/ovh/docs/pull/4373/commits/917d6f95d3e0f003df60f7a0c0e63eeb7ed5ce2b)

File: pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md (hunk 9 above)
Comment (philippart-s) [reply pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md]: done with [917d6f9](https://github.com/ovh/docs/pull/4373/commits/917d6f95d3e0f003df60f7a0c0e63eeb7ed5ce2b)

File: pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md (hunk 6 above)
Comment (philippart-s) [reply pages/platform/ai/dashboard_guide_01_getting_started/guide.en-gb.md]: done with [917d6f9](https://github.com/ovh/docs/pull/4373/commits/917d6f95d3e0f003df60f7a0c0e63eeb7ed5ce2b)

**Review by mhrng — APPROVED** [review]
Top, merci :)

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @philippart-s

**Pull request merged**